src/components/TopologySidebar/ComponentTeams.tsx

```diff
@@ -0,0 +1,60 @@
+import { AiOutlineTeam } from "react-icons/ai";
+import { useGetComponentsTeamQuery } from "../../api/query-hooks";
+import Title from "../Title/title";
+import CollapsiblePanel from "../CollapsiblePanel";
+import { Loading } from "../Loading";
+import EmptyState from "../EmptyState";
+import { Icon } from "../Icon";
+import { Link } from "react-router-dom";
+
+type Props = {
+  componentId: string;
+};
+
+export function ComponentTeams({ componentId }: Props) {
+  const { data: componentTeams, isLoading } = useGetComponentsTeamQuery(
+    componentId,
+    {
+      onError: (error) => {
+        console.error(error);
+      }
+    }
+  );
+
+  return (
+    <CollapsiblePanel
+      Header={
+        <div className="flex py-2 flex-row flex-1 items-center">
+          <Title
+            title="Teams"
+            icon={<AiOutlineTeam className="w-6 h-auto" />}
+          />
+        </div>
+      }
+    >
+      <div className="flex flex-col space-y-4 py-2 max-w-full">
```

Comment:
Why do we need to apply styles on the header and body?

Reply:
@moshloop removed the header styles, but body styles are needed to ensure vertical spacing (`space-y-4`) for the list of items.